.cascade-code/Chapman.edu/_cascade/formats/modular/uninav/uninav.vtl

```diff
@@ -294,7 +294,7 @@ On a page level, configure via:
 
       #if ( $isUmbrellaCategory )
         <span tabindex="0" class="toggle-menu-label">${umbrellaLabel} Menu</span>
-      #else
+      #elseif ( $currentPathRoot != "index" && $currentPathRoot != "")
```

Comment:
can you explain this?

Reply:
checking to make sure it's not the homepage which is just `index`

Reply:
It's already live on production